src/main/java/com/dumbdogdiner/stickyapi/bukkit/book/commonmarkextensions/ParserFormatExtension.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2020-2021 DumbDogDiner <dumbdogdiner.com>. All rights reserved.
+ * Licensed under the MIT license, see LICENSE for more information...
+ */
+package com.dumbdogdiner.stickyapi.bukkit.book.commonmarkextensions;
+
+import org.commonmark.parser.Parser;
+
+
+
+public class ParserFormatExtension implements Parser.ParserExtension {
+    private ParserFormatExtension(){}
+
+    public static ParserFormatExtension create() {
+        return new ParserFormatExtension();
+    }
+
+    @Override
+    public void extend(Parser.Builder parserBuilder) {
+        parserBuilder.customDelimiterProcessor(new ColorFormatDelimiterProcessor());
+        //parserBuilder.customDelimiterProcessor(new StrikethroughDelimiterProcessor());
+        /*parserBuilder.postProcessor(null);
+        parserBuilder.postProcessor(null);*/
+        // TODO add a parser here that does the line splitting
```

Comment:
So like, is this done or not?

Reply:
Book utils is, the markdown parsing is put on hold. I will split it to a separate branch shortly, I forgot tondo that.